lib/pluck_all/models/active_record_extension.rb

```diff
@@ -43,7 +43,10 @@ def to_sql_column_name
   else
     def pluck_all(*column_names, cast_uploader_url: true)
       column_names.map!(&to_sql_column_name)
-      return construct_relation_for_association_calculations.pluck_all(*column_names) if has_include?(column_names.first)
+      if has_include?(column_names.first)
+        relation = apply_join_dependency
+        return relation.pluck(*column_names)
```

Comment:
It should use `pluck_all` method here to get hash results.
```suggestion
        return relation.pluck_all(*column_names)
```
(https://github.com/khiav223577/pluck_all/compare/fix/test_pluck_all_with_includes)

Reply:
I've added some test cases to test this line. 
You could merge this branch [fix/test_pluck_all_with_includes](https://github.com/khiav223577/pluck_all/compare/fix/test_pluck_all_with_includes) to make sure if your modification fixes the bug or not :)

Reply:
@khiav223577 
Sorry for my careless mistake.
And thank you for your comments and adding some test cases for this patch.

Thanks to the test cases, I've found and fixed some bugs caused by differences of Rails versions.
Thank you very much.